src/main/java/frc/robot/Robot.java

```diff
@@ -142,9 +142,10 @@ public void robotPeriodic() {
   /** This function is called once when the robot is disabled. */
   @Override
   public void disabledInit() {
+    isUsingVision = true;
```

Comment:
Do we want to keep this change or keep vision off?

Reply:
i'll turn if off for comp.